heron/tmaster/src/cpp/manager/tmaster.cpp

```diff
@@ -364,7 +364,10 @@ proto::system::Status* TMaster::RegisterStMgr(
     // First check to see if that other guy has timed out
     if (!stmgrs_[stmgr_id]->TimedOut()) {
       // we reject the new guy
-      LOG(ERROR) << "Another stmgr exists with the same id and it hasn't timed out" << std::endl;
+      LOG(ERROR) << "Another stmgr exists at "
```

Comment:
Hmm, actually we already have a log for stmgr-id:

`LOG(INFO) << "Got a register stmgr request from " << stmgr_id << std::endl;`

So I don't think we need to log it again.